datarepo-clienttests/src/main/resources/logback.xml

```diff
@@ -4,7 +4,7 @@
             <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{50} - %msg%n</pattern>
         </encoder>
     </appender>
-    <root level="INFO">
+    <root level="DEBUG">
```

Comment:
Could you put this back to INFO before merging please? I think the verbose outputs can be confusing for people just starting to use the Test Runner.

Reply:
For sure! I'm keeping a list of things I need to do before merging in the [description](https://github.com/DataBiosphere/jade-data-repo/pull/622#issue-491195257).

Reply:
Oh duh. Sorry I missed that! Thank you!